test/server/lib/route-helpers.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+const moment = require('moment');
+const timeshift = require('timeshift');
+const stub = sinon.stub().resolves('finished');
+const { dateChecker, promiseResponder, dashboard, home } = require('proxyquire')('../../../src/lib/route-helpers', {
+	'./model': stub
+});
+
+describe('lib/routesHelpers', () => {
+	describe('dateChecker', () => {
+			describe('checks for invalid dates', () => {
+				it('should return false when the input is not a valid date', () =>
+					expect(dateChecker('2019-09-50'))
+						.to.equal('Invalid date'));
+			});
+			describe('checks for valid dates', () => {
+				it('should return an ISO formatted timestamp string', ()=> {
+					expect(dateChecker('2019-06-03'))
+						.to.be.a('string')
+						.and.to.equal('2019-06-03T00:00:00+01:00');
+					expect(dateChecker('2019-12-03'))
+						.to.be.a('string')
+						.and.to.equal('2019-12-03T00:00:00Z');
+				});
+			});
+		});
+
+	describe('promiseResponder', () => {
+		const next = sinon.stub(), render = sinon.stub(), send = sinon.stub();
```

Comment:
Could we split these across multiple lines; it took me a while to spot `send`'s declaration.

Reply:
fixed